dev_guide/event_notifications.adoc

```diff
@@ -0,0 +1,155 @@
+[[dev-guide-event-notifications]]
+= Event Notifications
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+For each of your projects, you can choose to receive email notifications about
+various xref:../dev_guide/events.adoc#dev-guide-events[events], including:
+
+* _Dead deployment_: A failed deployment in which no other deployment in the
+rollout history is able to remain active. The related `deploymentConfig` has no
+deployments with active replicas.
+
+* _Failed deployment_: A new deployment fails and another deployment in the
+ rollout history is now active. The related `deploymentConfig` has one active
+ deployment, but it is not the latest deployment.
+
+* _Dead build_: Occurs when automatically triggered builds are failing for a
+specified `buildConfig` that previously had successful builds.
+
+* _Dead persistentVolumeClaim_: A claim that is in the pending state for longer
+than some reasonable amount of time. These are claims whose resource requests
+likely do not match with any existing persistent volumes in the cluster.
+
+* _Lost persistentVolumeClaim_: A claim that is bound to a backing persistent
+volume, but the persistent volume has unexpectedly gone away.
+
+Notifications are sent to the email address of the creator of the project.
+
+[[event-notifications-how-it-works]]
+== How It Works
+
+A single thread analyzes the state of all objects and several object types in the
```

Comment:
This seems like it gets too far into the weeds with respect to our implementation. It's useful for other developers in the Online team, as well as our ops team, but not to users that simply want notifications.

Consider a more generic statement here, explaining that we continually monitor events across the cluster and issue notifications when problems are detected. It's also worth mentioning that we throttle notifications (so you won't get flooded with emails)